specs/differences-from-optimism-bedrock.md

```diff
@@ -23,33 +23,57 @@
 
 ## Nodes
 
-### Overview
+There are two types of network participants in the OP Stack:
 
-Kroma is composed of 3 nodes: `proposer`, `validator` and `vanilla`.  The following are components that
-are used to run each node:
+- [Sequencers](https://github.com/ethereum-optimism/optimism/blob/develop/specs/introduction.md#sequencers) consolidate
+ users' on/off chain transactions into blocks. They submit checkpoint outputs as well as batch transactions.
+- [Verifiers](https://github.com/ethereum-optimism/optimism/blob/develop/specs/introduction.md#verifiers) verify rollup
+   integrity and disputing invalid assertions.
 
-| Node        | Components                                                                          |
-|-------------|-------------------------------------------------------------------------------------|
-| `proposer`  | `L2 EL client` + `L2 CL client` + `kroma-batcher`                                   |
-| `validator` | `L2 EL client` + `L2 CL client` + `kroma-validator` + (optionally `kroma-prover`)   |
-| `vanilla`   | `L2 EL client` + `L2 CL client`                                                     |
+It is crucial to have at least one honest verifier who can verify the integrity of the rollup chain to ensure the
+ongoing security of the network. However, there exists a well-known challenge known as the 'Verifier's Dilemma' that
```

Comment:
IMO, the term `challenge` is confusing with our dispute challenge process. Wouldn't it be better to replace it with another term?
```suggestion
ongoing security of the network. However, there exists a well-known issue known as the 'Verifier's Dilemma' that
```

Reply:
what about 'obstacle'?

Reply:
Looks good!